Source/CarthageKit/Project.swift

```diff
@@ -106,23 +106,38 @@ public final class Project { // swiftlint:disable:this type_body_length
     
 	/// Whether to use authentication credentials from ~/.netrc file
 	/// to download binary only frameworks.
-	public var useNetrc = false
+    public var useNetrc: Bool {
+        get {
+            return binaryInstaller.useNetrc
+        }
+        set {
+            binaryInstaller.useNetrc = newValue
+        }
+    }
 
 	/// Sends each event that occurs to a project underneath the receiver (or
 	/// the receiver itself).
-	public let projectEvents: Signal<ProjectEvent, NoError>
-	private let _projectEventsObserver: Signal<ProjectEvent, NoError>.Observer
+    public var projectEvents: Signal<ProjectEvent, NoError>
```

Comment:
private (set) ?

Reply:
hm, I think it is some leftover, it can be `let` as it was.